db/user.go

```diff
@@ -38,7 +38,6 @@ type User struct {
 	Password    string
 	Gender      Gender
 	DateOfBirth time.Time
-	CreatedTime time.Time
```

Comment:
You did not fetch from remote!
do `git pull --all`

Reply:
i did.and it said you are up to date

Reply:
No problem, you need to integrate master branch with your branch, we'll discuss that later